src/garage/experiment/local_runner.py

```diff
@@ -132,7 +132,7 @@ def __init__(self, snapshot_config, max_cpus=1):
                                       last_path=None)
 
         self._algo = None
-        self._env = None
+        self.env = None
```

Comment:
_env

Reply:
It is used in off-policy algorithms base class, or is there other way we can have the `env` inside algorithms?

Reply:
oh ok. i think you can also use `runner.env`? if you are going to use the env, you should probably make a copy.

Reply:
Can you ask the sampler or runner for a copy of the environment? The algorithm has access to both of those.

Reply:
A copy of `gym` environment? From stackoverflow it seems impossible.

Reply:
new_env = `pickle.loads(pickle.dumps(env))`